Components/Modal/index.css

```diff
@@ -0,0 +1,48 @@
+.kash-modal {
+  display: flex;
+  flex-direction: column;
+  background-color: var(--white);
+  opacity: 1;
+  min-width: 30vw;
+  border-radius: 0.6rem;
+  top: -100vw;
+  position: relative;
+  /* min-height: 30vh; */
+}
+
+.kash-modal-header {
+  border-bottom: 1px solid rgba(0, 0, 0, 0.2);
+}
+.kash-modal-header > .fa-times-circle {
+  font-size: large;
+  cursor: pointer;
+}
+.kash-modal-body {
+  padding: 1rem;
```

Comment:
Use global variables for properties like margin, padding, and gap to maintain consistency across the website, something like --space-xsmall: 0.5rem

Reply:
ha i am using still forgetting sometimes slowly practicing